app.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+
+const app = express();
+const PORT = process.env.PORT;
+
+app.use(express.json());
+app.use(cors());
```

Comment:
일반적으로 cors를 미들웨어 상부에 위치시키는 것이 좋습니다.
```suggestion
app.use(cors());
app.use(express.json());
```